RELEASE_NOTES.md

```diff
@@ -6,6 +6,7 @@
 - fixed: checking whether a branch is merged to parent works for branches that have no common commit
 - added: CI/CD check ensuring that RELEASE_NOTES are up to date
 - fixed: `github create-pr` takes the PR title from the first unique commit
+- added: `--all`, `--mine`, `--by` flags and parameter `<PR-number-1> ... <PR-number-N>` in the `checkout-prs` subcommand
```

Comment:
```suggestion
- added: `--all`, `--mine`, `--by` flags and parameter `<PR-number-1> ... <PR-number-N>` to `git machete github checkout-prs`
```

Reply:
(since in the current shape it's not clear that it's about `github` command)

Reply:
Also! pls note that this PR constitutes a backward-compatible change to the CLI, which merits a MINOR version bump - 3.8.0

Reply:
Let's release the changes accumulated so far under 3.7.2 as 3.7.2, as suggested on Slack